automation_tools/satellite6/hammer.py

```diff
@@ -386,3 +386,21 @@ def attach_subscription_to_host_from_satellite(
         organization_id, product_name)
     return hammer('host subscription attach --subscription-id {0} '
                   '--host {1}'.format(subscription_id, hostname))
+
+
+def hammer_determine_cv_and_env_from_ak(ak_name, organization_id):
```

Comment:
@renzon , initially I thought to return tuple only. But I thought if anybody wants to use this function and dont want return both(one of them) he/she has to specify the index. And to find index he has to visit the docstring of function or function itself.